stable/mariadb/templates/pvc.yaml

```diff
@@ -0,0 +1,15 @@
+{{- if .Values.persistence.enabled }}
+---
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: {{ template "fullname" . }}
+  annotations:
+    volume.alpha.kubernetes.io/storage-class: {{ .Values.persistence.storageClass | quote }}
```

Comment:
Unfortunately the annotation did not change but the support did.

Check out the following PR to the docs:
https://github.com/kubernetes/kubernetes.github.io/pull/1311/files#diff-e3ed4a5817ba0330edba5ec941757bd0R416